src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', e => {
+  const thumbnail = e.target.closest('a');
```

Comment:
```suggestion
thumbs.addEventListener('click', e => {
  e.preventDefault();
  
  const thumbnail = e.target.closest('a');
```
do this at the beginning, not in the end